.azure-pipelines/generation-pipeline.yml

```
@@ -682,6 +687,35 @@ stages:
             repoName: msgraph-sdk-typescript
             packageName: '@microsoft/msgraph-sdk'
 
+- stage: stage_typescript_sdk_beta
+  dependsOn:
+  - stage_build_and_publish_kiota
+  - stage_beta_openapi
+  condition: |
+    and
+    (
+      eq(dependencies.stage_build_and_publish_kiota.result, 'Succeeded'),
+      in(dependencies.stage_beta_openapi.result, 'Succeeded', 'Skipped')
+    )
+  jobs:
+  - job: typescript_sdk_beta
+    steps:
+    - template: generation-templates/language-generation-kiota.yml
+      parameters:
+        language: 'typescript'
+        version: ''
```

Not a big deal, but it affects the title of the Generated PR to add info of the version.
```suggestion
        version: 'beta'
```